### PR TITLE
gh-97847: Fix misplaced or duplicated function descriptions in the multiprocessing documentation

### PR DESCRIPTION
**Pull request by geryogam**
* Issue: gh-97847

### CODE REVIEW

File: Doc/library/multiprocessing.rst [hunk 1]
```diff
@@ -116,9 +116,9 @@ to start a process.  These *start methods* are
 
   *forkserver*
     When the program starts and selects the *forkserver* start method,
-    a server process is started.  From then on, whenever a new process
+    a server process is spawned.  From then on, whenever a new process
     is needed, the parent process connects to the server and requests
-    that it fork a new process.  The fork server process is single
+    that it forks a new process.  The fork server process is single
```
Comment (merwok) [inline Doc/library/multiprocessing.rst]: This is an infinitive, _x requests that y do_ is a correct form.
```suggestion
    that it fork a new process.  The fork server process is single
```

Comment (geryogam) [reply Doc/library/multiprocessing.rst]: Thanks, updated.

**Comment by maggyero** [comment]
> Can you open a ticket to decribe the substantive problems to be fixed in the doc?
> (Likewise for your other PRs that change more than 10 lines)

Sure: https://github.com/python/cpython/issues/97847.

**Comment by erlend-aasland** [comment]
> > Can you open a ticket to decribe the substantive problems to be fixed in the doc?
> > (Likewise for your other PRs that change more than 10 lines)
> 
> Sure: #97847.

That issue describes stylistic changes to Sphinx/reST references only. However, this PR contains a load of other changes. That makes this PR very difficult to review; there is no explanation or rationale for most of the changes.

I therefore suggest closing this, together with the linked issue; it is highly unlikely to be reviewed, not to mention merged, in its current form.

**Comment by maggyero** [comment]
@erlend-aasland I have completely rewritten the issue as I realize it was very poorly explained.

**Comment by arhadthedev** [comment]
> @erlend-aasland I have completely rewritten the issue as I realize it was very poorly explained.

ping

**Comment by erlend-aasland** [comment]
I cannot see that my remark of https://github.com/python/cpython/pull/31482#issuecomment-1328347702 have been addressed; closing this PR.

If you want to follow up this, please open _targeted_ PRs for each change. As have been noted multiple times on your other PRs, please _avoid any stylistic or unrelated changes_.

**Pull request closed without merging**